analyze_data_with_r_using_posit_workbench_and_snowflake.md

```diff
@@ -0,0 +1,673 @@
+You can find Posit Workbench as a Snowflake Native Application
+and use it to connect to your database.
+
+#### Step 1: Navigate to Apps
+
+In your Snowflake account, Go to `Data Products` > `Apps` to open the Native Apps collection. If Posit Workbench is not already installed, click `Get`.
+
+![](assets/snowflake/05-get_posit_workbench.png)
+
+#### Step 2: Open the Posit Workbench Native App
+
+Once Posit Workbench is installed, click on the app under `Installed Apps` to launch the app. If you do not see the Posit Workbench app listed, ask you Snowflake account administrator for access to the app.
+
+![](assets/snowflake/06-open_app.png)
+
+After clicking on the app, you will see a page with configuration instructions and a blue `Launch app` button.
+
+![](assets/snowflake/09-launch-app.png)
+
+Click on `Launch app`. This should take you to the webpage generated for the Workbench application. You may be prompted to first login to Snowflake using your regular credentials or authentication method.
+
+### Create an RStudio Pro Session 
+
+Posit Workbench provides several IDEs, such as RStudio Pro, JupyterLab, and VS Code. For this analysis we will use an RStudio Pro IDE.
+
+#### Step 1: New Session
+
+Within Posit Workbench, click `New Session` to launch a new session to spin up your coding environment.
+
+![](assets/posit_workbench/01-start_new_session.png)
+
+#### Step 2: Select an IDE
+
+When prompted, select the RStudio Pro IDE.
+
+![](assets/posit_workbench/02-create_new_session_snowflake.png)
+
+#### Step 3: Log into your Snowflake account
+
+Next, connect to your Snowflake account from within Posit Workbench.
+Under `Credentials`, click the button with the Snowflake icon to sign in to Snowflake.
+Follow the sign in prompts.
+
+![](assets/posit_workbench/03-snowflake_login.png)
+
+When you're successfully signed into Snowflake, the Snowflake button will turn blue
+and there will be a checkmark in the upper-left corner.
+
+![](assets/posit_workbench/04-snowflake_login_success.png)
+
+#### Step 4: Launch the RStudio Pro IDE
+
+Click `Start Session` to launch the RStudio Pro IDE.
+
+Once everything is ready,
+you will be able to work with your Snowflake data
+in the familiar RStudio Pro IDE. Since the IDE is provided by the Posit Workbench Native App, 
+your entire analysis will occur securely within Snowflake.
+
+![](assets/rstudio/01-rstudio.png)
+
+#### Step 5: Access the Quickstart Materials
+
+This Quickstart will step you through the analysis contained in <https://github.com/posit-dev/snowflake-posit-quickstart-r/blob/main/quarto.qmd>.
+To follow along, open the file in your RStudio Pro IDE. There are two ways to do this:
+
+1. **Simple copy-and-paste** Go to File > New File > Quarto Document and then copy the contents of [quarto.qmd](https://github.com/posit-dev/snowflake-posit-quickstart-r/quarto.qmd) into your new file.
+2. **Starting a new project linked to the github repo.** To do this:
```

Comment:
Please add a comment that this will prompt for a passcode or how authentication works with Git since SSH is not enabled in SPCS.